orderer/consensus/etcdraft/eviction_test.go

```diff
@@ -194,6 +186,15 @@ func TestEvictionSuspector(t *testing.T) {
 			height:                     9,
 			halt:                       t.Fail,
 		},
+		{
+			description:                "our height is the highest",
```

Comment:
nit - this description probably should be updated to reflect actual semantics

Reply:
should this be back-ported to 1.4? (which i know is now EOL, but i saw commits are still being added to it...)

Reply:
I don't think this is important. You can get by without this fix.